docs/gathered-data.md

```diff
@@ -495,6 +497,19 @@ Response see:
   * 4.7+
 
 
+## OpenshiftLogging
+
+collects `logging.openshift.io` resources
+from Openshift Logging Stack.
+
+API Reference:
+  https://github.com/openshift/cluster-logging-operator/blob/21ba046939b664c2e7e2640dc20f5d063f4fffc2/pkg/apis/logging/v1/clusterlogging_types.go
```

Comment:
Sorry I probably provided some weird link. This one looks better IMO:
https://github.com/openshift/cluster-logging-operator/blob/master/pkg/apis/logging/v1/clusterlogging_types.go